canonical-web-and-design/vanilla-framework
# add react components doc

## Done

- Add React components docs generator script
- Add React component template
- Test it on the component accordion

## QA

- [demo](https://vanilla-framework-4369.demos.haus/docs/patterns/accordion)
- Make sure that the props and different variants are shown properly
- Delete the file: `react-components.json`
- Run the following script to regenerate the data:
```bash
yarn run generate-react-docs
```
- Make sure that the file `react-components.json` is created

## Screenshots

### example of react component documentation in the component accordion
![image](https://user-images.githubusercontent.com/36013798/159713636-6fcb4eb4-a445-4b67-84fb-6b5217f39d43.png)
